pkg/apis/triggers/v1alpha1/event_listener_types.go

```diff
@@ -227,12 +227,11 @@ func (els *EventListenerStatus) SetAddress(hostname string) {
 		els.Address = &duckv1alpha1.Addressable{}
 	}
 	if hostname != "" {
-		els.Address.Hostname = hostname
-		if u, err := apis.ParseURL(fmt.Sprintf("http://%s/", hostname)); err != nil {
-			els.Address.URL = u
+		els.Address.URL = &apis.URL{
+			Scheme: "http",
```

Comment:
So, the status now only has Adress.URL and no Address.Hostname. I think this is totally fine. Just worth calling this out as a breaking change in the `Release Notes` section of the PR!